client/src/containers/Setting/Setting.js

```diff
@@ -12,7 +12,7 @@ class Setting extends Component {
     render() {
         var localLang = localStorage.getItem("language");
         return (
-            <div className="row">
+            <div className="row app-select-language">
```

Comment:
I don't understand what you changed here - how the does Language Selector look like now?